### ISSUE TITLE
[WebApplication.java] Soucis de faute de frappe dans un commentaire

### ISSUE THREAD

**Issue by dokor** [Coreoz/Plume-archetypes#9]
Dans l'archétype "classique" de plume, on retrouve un fichier WebApplication.java. 
Ce dernier comporte en son centre un commentaire qui contient une erreur de frappe qui est visible lorsqu'on décommente le code lors de l'utilisation de plume-scheduler. 

```
			// Add a shutdown hook to execute some code when the JVM receive a kill signal before it stops
			addShutDownListener(httpServer);
			// If Plume Scheduler / Wisp is used, uncomment next line
			// addShutDownListerner(httpServer, injector.getInstance(Scheduler.class));
```

Il y a un R en trop dans `// addShutDownListeRner(XXXXXX);`

(Antoine LE LOUËT)

**Comment by amanteaux** [comment]
Normalement c'est déjà corrigé dans la release [4.2.0](https://github.com/Coreoz/Plume-archetypes/commit/4ee4a97fa18514f9183e1d156002ac5fbc7fc3e3).

Le commit de correction est celui là : 7623f1c8e6e2ed9c1dfe9f9f9bafb3c98731e7c4

Tu reproduits le problème quand tu crées un projet avec la commande décrite dans https://github.com/Coreoz/Plume-archetypes/tree/master/plume-archetype-querydsl-jersey-guice-grizzly ?

**Comment by dokor** [comment]
Yes j'ai bien utilisé la commande normal qui est décrite dans la doc

**Comment by dokor** [comment]
HA ! 

Le commit en question corrige bien le nom de la méthode et de son utilisation, mais la modif n'a pas été impacté sur le commentaire de la ligne 49 justement. 
Ce qui fait que lorsqu'on décommente, la fonction n'est pas trouvé.

**Comment by amanteaux** [comment]
Ah oui effectivement, j'avais pas compris, c'est la ligne commentée qui pose problème !

Je note ça pour la release qui arrive normalement très bientôt !

**Comment by amanteaux** [comment]
Merci pour l'ouverture de l'issue @dokor !

**Comment by amanteaux** [comment]
Ca a été livré dans la release 5.0.0, j'avais oublié de mettre ce ticket en clos
